packages/web/src/components/track/DownloadSection.tsx

```diff
@@ -353,16 +310,13 @@ export const DownloadSection = ({ trackId }: DownloadSectionProps) => {
                     ? STEM_INDEX_OFFSET_WITH_ORIGINAL_TRACK
                     : STEM_INDEX_OFFSET_WITHOUT_ORIGINAL_TRACK)
                 }
-                isOriginal={quality === DownloadQuality.ORIGINAL}
+                isOriginal
```

Comment:
is it always the case that isOriginal is true for the instances of this component? if so can we further simplify DownloadRow? i suppose same goes for size/fileSizes?

Reply:
good call, we can remove the prop from `DownloadRow` completely now

Reply:
and `filename`